### PR TITLE
New doc: Understanding Animations

### PR DESCRIPTION
**Pull request by trekladyone**
Document extracted from the original Introduction document, which is to be retired. Linting suggested corrections made.  Restructuring or rewording sentences to be shorter and clearer.

### CODE REVIEW

**Comment by jessicajaniuk** [comment]
@trekladyone It looks like you assigned this to me. Did you mean to request my review?

**Comment by trekladyone** [comment]
Hi, Jessica.

 

Yes, I requested your review on all the new Animations documents.

File: aio/content/guide/understanding-angular-animation.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+In this code snippet, multiple style attributes are set at the same time for the state.
+In the `open` state, the button has a height of 200 pixels, an opacity of 1, and a yellow background color.
+
+<code-example header="src/app/open-close.component.ts" path="animations/src/app/open-close.component.ts" region="state1"></code-example>
+
+In the following `closed` state, the button has a height of 100 pixels, an opacity of 0.8, and a background color of blue.
+
+<code-example header="src/app/open-close.component.ts" path="animations/src/app/open-close.component.ts" region="state2"></code-example>
+
+## Animation transition and timing
```
Comment (jessicajaniuk) [inline aio/content/guide/understanding-angular-animation.md:68]: Isn't this content supposed to be on a separate new doc defined in https://github.com/angular/angular/pull/47909?

File: aio/content/guide/understanding-angular-animation.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+ng g component open-close
+
+</code-example>
+
+This creates the component at `src/app/open-close.component.ts`.
+
+</div>
+
+## Animation state and style
```
Comment (jessicajaniuk) [inline aio/content/guide/understanding-angular-animation.md:47]: Is this content supposed to be here?  Or is it supposed to be referencing the new file with this content in https://github.com/angular/angular/pull/47912?

File: aio/content/guide/understanding-angular-animation.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+In this example, name the trigger `openClose`, and attach it to the `button` element.
+The trigger describes the open and closed states, and the timings for the two transitions.
+
+<div class="alert is-helpful">
+
+**NOTE**: <br />
+Within each `trigger()` function call, an element can only be in one state at any given time.
+It is possible, though, for multiple triggers to be active at once.
+
+## Define an animation
```
Comment (jessicajaniuk) [inline aio/content/guide/understanding-angular-animation.md:188]: This also looks to be the content in the new doc in #47912. Is it intended to duplicate that content here in this doc?

File: aio/content/guide/understanding-angular-animation.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+    <code-example format="typescript" language="typescript">
+
+    transition( 'on =&gt; off, off =&gt; void' )
+
+    </code-example>
+
+</div>
+
+## Trigger an animation
```
Comment (jessicajaniuk) [inline aio/content/guide/understanding-angular-animation.md:171]: I think this section not really covered by other documents.

**Review by jessicajaniuk — COMMENTED** [review]
I'm a little confused with this document and could use some clarification. The sections here mostly look to be duplicated from other docs, and I'm mainly confused why we're doing that on this page. Could you clarify that? Wouldn't we want to avoid duplication of content?

**Review by jessicajaniuk — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit bca76ed43aff90c4cd55fc515c14fef359a60b26.

**Pull request closed without merging**